neuvector-scanner.yaml

```diff
@@ -59,6 +61,33 @@ subpackages:
           mkdir -p ${{targets.contextdir}}/usr/local/bin
           install -Dm755 monitor/monitor ${{targets.contextdir}}/usr/local/bin/monitor
 
+  - name: neuvector-db
+    description: NeuVector vulnerability database for the SUSE NeuVector Container Security Platform
+    pipeline:
+      - runs: |
+          # Fetch NeuVector scanner from docker hub
+          # https://hub.docker.com/r/neuvector/scanner/tags
+          # we don't use the dbgen tool for this since it's very slow and
+          # super flaky. Extracting is faster and is more reliable.
+          # This may change in the future if neuvector isn't published to DH
+          crane export neuvector/scanner:${{package.version}} scanner.tar
+
+          # Extract image
+          mkdir -p scanner
+          tar xvf scanner.tar -C scanner
+
+          # Install DB
+          mkdir -p ${{targets.contextdir}}/etc/neuvector/db
+          cp scanner/etc/neuvector/db/cvedb ${{targets.contextdir}}/etc/neuvector/db/cvedb
+    test:
+      pipeline:
+        - runs: |
+            if [[ -f "/etc/neuvector/db/cvedb" ]]; then
+              echo "CVE DB found!" && exit 0
+            else
+              echo "CVE DB not found!" && exit 1
+            fi
```

Comment:
Another nit, but in addition to this test, extending the top-level `test:` with something like `scanner -v -d /etc/neuvector/db/` might give us a legit test of the database; I wasn't able to confirm this works (because I can't currently install the DB :sweat_smile:), but it does emit:

```
Error: Read db file fail: open /etc/neuvector/db/cvedb: no such file or directory
```
and `--help` says:
```
  -v	show cve database version
```
so it might work as a standalone test.